### PR TITLE
Fix #4075  - Replace Embed property with EmbeddedFiles item group in Microsoft.FSharp.Targets

### LINKED ISSUE #4075: Replace Embed property with EmbeddedFiles item group in Microsoft.FSharp.Targets
We need to unify on ```EmbeddedFiles``` item group accross C#, VB and F#. The former two use ```EmbeddedFiles``` item group while F# uses ```Embed``` property.

Replace

```
Embed="$(Embed)"
```

with

```
Embed="@(EmbeddedFiles)"
```

### PR DESCRIPTION
**Pull request by KevinRansom**
FSharp currently uses the Embed collection to include source files.  

This switches to EmbeddedFiles, it enables Embed for back compat.

### CODE REVIEW

File: src/fsharp/FSharp.Build/Microsoft.FSharp.Targets [hunk 1]
```diff
@@ -172,6 +172,10 @@ this file.
 
     <PropertyGroup>
         <CoreCompileDependsOn>_ComputeNonExistentFileProperty</CoreCompileDependsOn>
+
+        <!-- Provided for compatability with the initial releases of FSharp for the dotnet sdk -->
+        <!-- New project files should use 'EmbeddedFiles' -->
+        <EmbeddedFiles Condition=" '$(Embed)' != '' ">$(EmbeddedFiles);$(Embed)</EmbeddedFiles>
```
[the remark this answers is not in the document]
Comment (KevinRansom) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.Targets]: collection

[the remark this answers is not in the document]
Comment (brettfo) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.Targets]: Should it not be `@(EmbeddedFiles)`, then?

[the remark this answers is not in the document]
Comment (KevinRansom) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.Targets]: @brettfo yes I think it should, thx.

**Comment by KevinRansom** [comment]
@brettfo much better fix, also updated the Build task too.

**Pull request merged**